sql/core/src/test/scala/org/apache/spark/sql/execution/BenchmarkWholeStageCodegen.scala

```diff
@@ -41,20 +42,20 @@ class BenchmarkWholeStageCodegen extends SparkFunSuite {
 
     benchmark.addCase("Without codegen") { iter =>
       sqlContext.setConf("spark.sql.codegen.wholeStage", "false")
-      sqlContext.range(values).filter("(id & 1) = 1").count()
+      sqlContext.range(values).filter("(id & 1) = 1").groupBy().sum().collect()
     }
 
     benchmark.addCase("With codegen") { iter =>
       sqlContext.setConf("spark.sql.codegen.wholeStage", "true")
-      sqlContext.range(values).filter("(id & 1) = 1").count()
+      sqlContext.range(values).filter("(id & 1) = 1").groupBy().sum().collect()
     }
 
     /*
       Intel(R) Core(TM) i7-4558U CPU @ 2.80GHz
-      rang/filter/aggregate:            Avg Time(ms)    Avg Rate(M/s)  Relative Rate
+      rang/filter/aggregate:             Avg Time(ms)    Avg Rate(M/s)  Relative Rate
       -------------------------------------------------------------------------------
-      Without codegen             7775.53            26.97         1.00 X
-      With codegen                 342.15           612.94        22.73 X
+      Without codegen                         5488.16            38.21         1.00 X
+      With codegen                             531.08           394.88        10.33 X
```

Comment:
The benchmark is not that stable, this number change from 10 to 20, maybe 200M are still not enough, I will increase it to 500M.